paasta_tools/contrib/shared_ip_check.py

```diff
@@ -33,7 +50,19 @@ def main():
         return 2
     else:
         print('OK - No Docker containers sharing an IP on this host.')
-        return 0
+
+    seen = []
```

Comment:
This could be a dict of target_parameters to rules, then checking if it contains target_parameters would be faster and you don't need find_rule.

Reply:
Ah! good idea